modules/rostests/win32/user32/kbdlayout/kbdlayout.c

```diff
@@ -13,7 +12,7 @@
 
 
 
-LRESULT MainDialogProc(HWND hDlg, UINT Msg, WPARAM wParam, LPARAM lParam);
+INT_PTR WINAPI MainDialogProc(HWND hDlg, UINT Msg, WPARAM wParam, LPARAM lParam);
```

Comment:
```suggestion
INT_PTR CALLBACK MainDialogProc(HWND hDlg, UINT Msg, WPARAM wParam, LPARAM lParam);
```

Logically, `MainDialogProc` is not a part of Windows API.